spec/fixtures/unit/puppetfile/various-modules/modules/apt/useless_content

```diff
@@ -0,0 +1 @@
+This only exists so the directory can be commited to git for testing purposes.
```

Comment:
committed has two 't's ; P

Reply:
I think the convention is also usually to call this file `.gitkeep`.

Reply:
TIL

Reply:
about `.gitkeep`, not my spelling deficiencies.